package.json

```diff
@@ -2,12 +2,15 @@
   "name": "mgrs",
   "version": "1.0.1-alpha",
   "description": "Utility for converting between WGS84 lat/lng and MGRS coordinates",
-  "main": "dist/mgrs.js",
-  "module": "mgrs.js",
+  "main": "dist/mgrs.min.js",
+  "module": "dist/mgrs.esm.js",
+  "types": "index.d.ts",
   "scripts": {
     "lint": "eslint mgrs.js && eslint test",
-    "test": "npm run build && nyc mocha test/test.js",
-    "build": "mkdir -p dist && rollup -c"
+    "pretest": "npm run clean && npm run build",
+    "test": "nyc mocha test/test.js",
+    "clean": "rimraf dist",
+    "build": "mkdirp dist && rollup -c"
```

Comment:
Could you provide more context on why the switch to `mkdirp` from `mkdir -p`?  Is there any alternative to mkdirp?  Unfortunately, it doesn't look like the project is still being maintained: https://www.npmjs.com/package/mkdirp
https://www.npmjs.com/package/mkdirp

Reply:
`mkdir -p` on Windows makes a directory called "-p".  I wish I were kidding.

I didn't bother researching to see if there's a maintained project for this, and honestly it's a tiny chunk of functionality so it might be simpler to just write a 4-line script that does it using `fs` methods, but I don't know of a cross-platform way to do this using shell commands.

Reply:
I think we can just change `mkdirp` to `mkdir` and removed the `mkdirp` dependency.  I don't think the `-p` is necessary.  Let me know if I missed anything!

Reply:
I replaced `mkdirp` with `mkdir`.  Let me know if this doesn't work for some reason.